packages/react-native-bridge/android/react-native-bridge/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,9 @@
         </intent>
     </queries>
 
+    <!-- Vibrate permission required for haptic feedback -->
+    <uses-permission android:name="android.permission.VIBRATE" />
+
```

Comment:
This is required by the vibration library on Android. I'd like to note that this permission is already included in the WordPress Android app ([reference](https://github.com/wordpress-mobile/WordPress-Android/blob/1ff769e329eedcda49cee15869c7d6f859091f25/WordPress/src/main/AndroidManifest.xml#L13)).